proxy/config/struct.go

```diff
@@ -66,3 +67,19 @@ type Admin struct {
 	Enable    bool   `yaml:"enable"`
 	ServerURI string `yaml:"serverUri"`
 }
+
+//Tracing has attributes for APM
+type Tracing struct {
+	Enable    bool   `yaml:"enable"`
+	ServerURI string `yaml:"serverUri"`
+}
+
+//Apm is for Application Performance Management
+type Apm struct {
```

Comment:
all upper case APM

Reply:
Modify according to code review comments